src/libraries/System.Private.CoreLib/src/System/Index.cs

```diff
@@ -4,6 +4,7 @@
 
 using System.Diagnostics;
 using System.Runtime.CompilerServices;
+using System.Text;
```

Comment:
Is this addition intentional?

Reply:
It was needed originally when I was using a StringBuilder for ToString, but I removed that. Will revert.